openreview/stages/arr_content.py

```diff
@@ -1889,7 +1889,10 @@
 del arr_sac_max_load_task["maximum_load_this_cycle_for_resubmissions"]
 arr_sac_max_load_task["maximum_load_this_cycle"] = {
     "value": {
-        "param": {"regex": "[0-9]{0,3}", "optional": False, "type": "string"}
+        "param": {
```

Comment:
is there a test that uses this invitation?

Reply:
Yes it gets tested with the other max load invitations:
https://github.com/openreview/openreview-py/blob/7f2eb6729a79325aed6349efc6cbacd30a6d5fba/tests/test_arr_venue_v2.py#L1463-L1471

Reply:
don't you need to pass 10 as number instead of a string?